newsletters/templates/newsletters/detail.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'tgrsite/addnavbar.html' %}
+{% load markdown_tags %}
+{% load static %}
+
+{% block title %}{{object.title}}{% endblock %}
+{% block leftcontents %}
+{% if object.author == user.member or not object.author %}
+	<p><a href="{% url 'newsletters_update' object.id %}">Edit Newsletter</a></p>
+	<p><a href="{% url 'newsletters_delete' object.id %}">Delete Newsletter</a></p>
+	{% if object.ispublished %}<p><a href="{% url 'newsletters_email' object.id %}">Email Version (go to this page and hit Ctrl-S to save a version to distribute as an email!)</a></p>{% endif %}
+{% endif %}
+{% endblock %}
+
+{% block body %}
+
+<img src={% static 'newsletter-banner.png' %} width='100%' />
+{% if not object.ispublished %}<h1><strong>[UNPUBLISHED]</strong></h1>{% endif %}
+<h1>{{object.title}}
+	<small>{% if object.author %} by <a href='{% url 'user' object.author.id %}'>{{object.author}}</a> {% endif %}<small>({{object.pub_date}})</small></small></h1>
```

Comment:
Double quotes for HTML props
`<a href="{% ... %}">`

`</small>` is doubled up

Reply:
Fixed.

Reply:
Small double up is intentional - have <small>author <small>date</small></small>